UI/window-projector.cpp

```diff
@@ -350,82 +353,98 @@ void OBSProjector::OBSRenderMultiview(void *data, uint32_t cx, uint32_t cy)
 	auto calcBaseSource = [&](size_t i)
 	{
 		switch (multiviewLayout) {
+		case MultiviewLayout::HORIZONTAL_TOP_24_SCENES:
+			window->sourceX = (i % 6) * window->scenesCX;
+			window->sourceY = window->pvwprgCY +
+					(i / 6) * window->scenesCY;
+			break;
 		case MultiviewLayout::VERTICAL_LEFT_8_SCENES:
-			window->sourceX = window->halfCX;
-			window->sourceY = (i / 2 ) * window->quarterCY;
+			window->sourceX = window->pvwprgCX;
```

Comment:
Unfortunately, due to the name change of these variables, this commit also appears to be affected with sort of "unrelated" changes, or so it feels.  Perhaps I will need to consult with you on why the variable name changes occurred, but they don't appear to have been mentioned in any commit messages.